content/blog/2021-11-30-november-21-community-gems.md

```diff
@@ -185,7 +185,7 @@ Thanks for this question @evergreengt!
 
 ---
 
-https://media.giphy.com/media/jS27LWasgUIYrXtP83/giphy.gif
+https://media.giphy.com/media/VInc9GYelUbHf5QhNR/giphy.gif
```

Comment:
The original gif was someone saying "accomplished". So I just clicked on one of the first gifs that popped up when you searched "accomplished" and replaced it.